src/components/Charts/EmptyPanel.tsx

```diff
@@ -0,0 +1,18 @@
+/** Renders in place of a chart when there is no data */
+
+import React from 'react';
+import { EmptyPanelContainer } from 'components/Charts/Charts.style';
+
+const EmptyPanel: React.FC<{ bodyText: React.ReactElement }> = ({
+  bodyText,
+  children,
+}) => {
+  return (
+    <EmptyPanelContainer>
+      {bodyText}
+      {children}
+    </EmptyPanelContainer>
+  );
+};
```

Comment:
We do we need the `bodyText` as a separate prop? we could just use `children` for that right?

Reply:
mm yes true

Reply:
originally it was just the `bodyText` prop, but then i noticed that the one in Explore.tsx takes `ScreenshotReady`, so i added children. sidenote- does the empty panel need `ScreenshotReady`?

Reply:
we might as well include it to cover the case where someone shares an empty tab (the tab could become empty after being shared, if that specific dataset has problems one day)